### DISCUSSION TITLE
how on earth do i even fix the ip

### DISCUSSION THREAD

**Discussion by randogaming6252** [Palakis/obs-websocket#898] [Q&A]
my Ip is reading as gibberish and it needs to be corrected, but due to the lack of any form of clear instruction i am now here after breaking the install 18 separate times and having to reinstall. any form of help would be great. if there's no way to set the Ip manually then that needs to be a feature

**Comment by tt2468** [comment]
obs-websocket binds to all interfaces (aka 0.0.0.0). You just need to connect on one of your computer's valid interfaces.
